### ISSUE TITLE
TypeError: require.extensions.hasOwnProperty is not a function at requireDir

### ISSUE THREAD

**Issue by dvera123** [ferreiratiago/gulp-git-push#3]
node: v8.1.4
npm: 5.0.3

I got this issue specific in gulp-git-push, any thoughts?

**Comment by huixisheng** [comment]
the temporary solution is 
```
$ cd node_modules/gulp-git-push/node_modules/gulp-git 
$ sudo npm i require-dir@0.3.2 -D
```

please see https://github.com/aseemk/requireDir/issues/45

**Comment by ferreiratiago** [comment]
A new version has been published `+ gulp-git-push@2.0.0`.
